src/MyParcelComApi.php

```diff
@@ -1148,7 +1148,7 @@ public function doRequest(
         array $headers = [],
         $ttl = self::TTL_NO_CACHE,
     ): ResponseInterface {
-        if (!str_starts_with($uri, $this->apiUri)) {
+        if (!str_starts_with($uri, 'http')) {
```

Comment:
Since all links that need this prepend functionality are absolute, we could do this instead:
```suggestion
        if (str_starts_with($uri, '/')) {
```